src/components/one-time-donation/Steps.tsx

```diff
@@ -65,9 +67,13 @@ export default function DonationStepper() {
     try {
       const data = {
         currency: 'BGN',
-        priceId: values.amount,
+        priceId: values.amount !== 'other' ? values.amount : undefined,
+        amount:
+          values.amount === 'other'
+            ? Math.round((values.otherAmount + Number.EPSILON) * 100)
```

Comment:
@borislavstoychev What's the goal here using `Number.EPSILON` ?

Reply:
We take the number and add a very small number, ```Number.EPSILON``` , to ensure the number’s accurate rounding.